third_party/move/move-compiler-v2/src/pipeline/uninitialized_use_checker.rs

```diff
@@ -216,11 +212,24 @@ impl UninitializedUseChecker {
                                 Initialized::Maybe => "possibly ",
                                 _ => "",
                             },
-                            target.get_local_name_for_error_message(*src)
+                            target.get_local_name_for_error_message(*temp)
                         ),
                     );
                 }
-            });
+            };
+            if let Bytecode::SpecBlock(_, spec) = bc {
+                for cond in &spec.conditions {
```

Comment:
Is `conditions` the only field of `spec` that needs to be checked? How about `update_map`?

Reply:
Yes, `update_map` is not yet filled in this phase AFAIK. The test case with `update z = y;` is used to show it.

Reply:
Maybe we can just say that in a comment here, so that it is explicit to the reader.

Reply:
done